Composer/packages/lib/shared/src/featureFlagUtils/index.ts

```diff
@@ -23,9 +23,8 @@ export const getDefaultFeatureFlags = (): FeatureFlagMap => ({
   },
   ORCHESTRATOR: {
     displayName: formatMessage('Orchestrator'),
-    description: formatMessage(
-      'Use as intent-only recognizer, typically for routing to skills or subsequent LUIS or QnAMaker processing or when entity extraction is not needed.'
-    ),
+    description: formatMessage('Recognize an intent, and take action or route  to a skill, LUIS app, or QnAMaker KB.'),
```

Comment:
Looks good to me except that there is an extra space in the `formatMessage` string b/w the words "route" and "to":
'Recognize an intent, and take action or **route  to** a skill, LUIS app, or QnAMaker KB.'

Reply:
Thanks, great eye! Fixed.